eZ/Publish/Core/Persistence/Legacy/Tests/Content/FieldValue/Converter/DateAndTimeTest.php

```diff
@@ -256,10 +256,14 @@ public function testToFieldDefinitionCurrentDate()
         );
 
         $this->converter->toFieldDefinition($storageDef, $fieldDef);
+        sleep(2);
```

Comment:
There is [this](https://www.schmengler-se.de/en/2011/03/php-mocking-built-in-functions-like-time-in-unit-tests/) and [this](https://github.com/php-mock/php-mock-phpunit), but I don't think they will work here as test and code use different namespaces.